cmd/site-replication.go

```diff
@@ -548,6 +552,7 @@ func (c *SiteReplicationSys) AddPeerClusters(ctx context.Context, psites []madmi
 		Name:                    sites[selfIdx].Name,
 		Peers:                   joinReq.Peers,
 		ServiceAccountAccessKey: svcCred.AccessKey,
+		ReplicateILMExpiry:      opts.ReplicateILMExpiry,
 	}
```

Comment:
need to pass the ilm expiry setting to. SRPeerJoin

Reply:
Ok